estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class EstatePropertyTag(models.Model):
+    _name = "estate.property.tag"
+    _description = "Estate Property Type"
+    _order = "name"
+
+    name = fields.Char(required=True)
+    color = fields.Integer(default=1)
```

Comment:
Could you improve to have random colors? So that anytime I add a tag it has a different color :rainbow: